### DISCUSSION TITLE
Nested taxonomy workaround?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6080] [General]
Hi, 

I've got a `collection` of products that would benefit from taxonomy nesting, but that doesn't exist in `3.3.12`. 

- I understand a possible workaround is to use a second collection, `collection:categories` to replace taxonomies altogether. 
- The idea is to use an `entries_field` that uses `collection:categories`  in the `collection:products` blueprint

On `categories/index` this is used to show a nested list of 'categories'
```
   <ul>
      {{ nav:collection:categories }}
        <li>
            <a href="{{ route name="cat.show" :slug="slug" }}">{{ title }}</a>
               {{ if children }}
                  <ul>{{ *recursive children* }}</ul>
               {{ /if }}
         </li>
      {{ /nav:collection:categories }}
  </ul>
```

### The problem

- this is on `categories/show` 
- `cat_entries_field ` is the name of `collection:categories` entries field
- The code below shows every entry that uses `cat_entries_field `
- I'm trying to filter `collection:products` by the `slug`

```
    {{ collection:products as="entries"}}
           <ul> 
              {{ entries }}
                  {{ if cat_entries_field }}
                      <li><a href="{{ url }}">{{ title }}</a></li>
                  {{ /if }}
              {{ /entries }}
          </ul>                
      {{ /collection:products }}
 ```

**Comment by ryanmitchell** [comment]
`{{ collection:products as="entries" :cat_entries_field:contains="page:id" }}`
